### ISSUE TITLE
Add ability to resize the iframe.

### ISSUE THREAD

**Issue by kabadesu** [Malvid/Malvid#28]
It would be pretty cool if you could supply a set of breakpoints in the config that allowed users to see each component at those different breakpoints.

And/or provide a way to resize the iframe manually.

**Comment by electerious** [comment]
I liked the idea at the first glance, but I'm not a fan the more I think about it:

1) A set of breakpoints leads to testing on a set of breakpoints, but all sizes are equal relevant. Components should be tested responsive on a range of widths.
2) The build-in responsive mode of Chrome and Safari is pretty good and we aren't able to provide the same functionality. So a better way to test is always just one keystroke away.

**Comment by kabadesu** [comment]
I do agree with your points - The purpose of adding the breakpoint buttons is to enable non-developers to view components at agreed upon breakpoints (clients).Also, our designers aren't afraid of the development tools, but it would make there jobs fractionally easier if they were able to  check our components against the breakpoints they designed for.

**Comment by emsuiko** [comment]
I agree with @kabadesu , it would be non-dev (read: customer/client) friendlier to have the ability to switch breakpoints (e.g. mobile, tablet, desktop) via button/slider.
I saw it some time ago at the sipgate design pattern library: https://www.sipgatedesign.com/pattern-library/agenda
Would be grand if something like that could be added to Malvid. Maybe I/we can help with that somehow?

**Comment by electerious** [comment]
I agree. The breakpoint feature could be indeed nice for the client and designer. Here's what it needs to implement it:

1. The iframe is currently not resizable (only the sidebar). It needs another handle to change the size.
2. There must be a new button on the top right toolbar that opens a dropdown with available breakpoints.
3. Breakpoints wider than the current resolution must be inactive in the dropdown.
4. There must be a few pre-configured breakpoints.
5. It must be possible to overwrite the pre-configured breakpoints via the config. The config should be an array containing objects. Each object should provide a width in px and a title (for the dropdown).
6. Documentation must be updated
